src/blog/2024/03/using_kafka_with_node-red.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Using Kafka with Node-Red - Part 1
+subtitle: Getting started with Kafka.
+description: Getting started with Kafka.
+date: 2024-03-26
+authors: ["sumit-shinde"]
+image: /blog/2024/03/images/using-kafka-with-node-red.png
```

Comment:
```suggestion
title: How Kafka is applied in manufacturing
subtitle: An overview of Kafka -- How it's applied for industrial applications, and how it works
description: An overview of Kafka -- How it's applied for industrial applications, and how it works
date: 2024-03-15
authors: ["sumit-shinde"]
image: /blog/2024/03/images/using-kafka-with-node-red.png
```

Tile would need updating

Reply:
I second this.  There isn't much discussion of Node-RED in the article, which is fine.  Just don't want to mislead people.

Reply:
@Yndira-E Can you update the image to match this new title please?